src/epgsql.erl

```diff
@@ -243,3 +243,12 @@ sync_on_error(C, Error = {error, _}) ->
 sync_on_error(_C, R) ->
     R.
 
+send_or_terminate(Connection, Request) ->
+    gen_server:call(Connection, Request, infinity).
+    %case gen_server:call(Connection, Request, ?EXEC_MAX_WAIT) of
```

Comment:
...the alternative interpretation of the state of things would be: you've gone with changing _this_ module, but then realised that the action should happen in `epgsql_sock`... 🤔